protos/sources/vega/markets.proto

```diff
@@ -22,31 +22,31 @@ message AuctionDuration {
 // Future product definition
 message Future {
   // The asset for the future
-  string settlement_asset = 2;
+  string settlement_asset = 1;
   // Quote name of the instrument
-  string quote_name = 4;
-  // The oracle spec describing the settlement data oracle filter
-  oracles.v1.OracleSpec oracle_spec_for_settlement_data = 5;
+  string quote_name = 2;
+  // The data source specification that describes the settlement data source filter
+  vega.data.v1.DataSourceSpec data_source_spec_for_settlement_data = 3;
 
-  // the oracle spec describing the trading termination oracle filter
-  oracles.v1.OracleSpec oracle_spec_for_trading_termination = 6;
+  // The data source specification that describes the trading termination data source filter
+  vega.data.v1.DataSourceSpec data_source_spec_for_trading_termination = 4;
 
-  // The binding between the oracle spec and the oracle data
-  OracleSpecToFutureBinding oracle_spec_binding = 7;
+  // The binding between the data spec and the data source
+  DataSourceSpecToFutureBinding data_source_spec_binding = 5;
 
-  // The number of decimal places implied by the settlement data (such as price) emitted by the settlement oracle
-  uint32 settlement_data_decimals = 8;
+  // The number of decimal places implied by the settlement data (such as price) emitted by the settlement data source
+  uint32 settlement_data_decimals = 6;
 }
 
-// OracleSpecToFutureBinding tells on which property oracle data should be
+// DataSourceSpecToFutureBinding tells on which property of the data source data should be
 // used as settlement data and which to use the trading terminated trigger
```

Comment:
```suggestion
// DataSourceSpecToFutureBinding describes which property of the data source data is to be
// used as settlement data and which to use as the trading terminated trigger
```

Reply:
@candida-d , all is addresses - separated the changes in a separate commit here -> https://github.com/vegaprotocol/vega/pull/5980/commits/7a15c5b1865961033f0128d64f9cefd2309f8851
for you to follow up.
I'll then squash it in the main one
Thanks!